packages/common/core-interfaces/src/fluidLoadable.ts

```diff
@@ -17,11 +17,16 @@ export interface IProvideFluidLoadable {
 	readonly IFluidLoadable: IFluidLoadable;
 }
 /**
- * A shared FluidObject has a URL from which it can be referenced
+ * A shared {@link FluidObject} has a URL from which it can be referenced.
```

Comment:
This is stale from when it used to have a `url` member.
```suggestion
 * A shared {@link FluidObject} has a handle which can be used to retrieve it.
```

Reply:
Should this also say "_which_ has a handle..."?